presto-docs/src/main/sphinx/release/release-323.rst

```diff
@@ -0,0 +1,50 @@
+===========
+Release 323
+===========
+
+General Changes
+---------------
+
+* Fix query failure when referencing columns from table containing hidden
+  columns. (:issue:`1796`)
+* Fix rare failure on the client side due to the server sending incorrect
```

Comment:
> Fix a rare issue in which the server produces an extra row containing the boolean value `true` as the last row in the result set. For most queries, this will result in a client error, since this row does not match the result schema, but is a correctness issue when the result schema is a single boolean column.